### DISCUSSION TITLE
Support for equalizer apps like Wavelet

### DISCUSSION THREAD

**Discussion by guilhermequinta** [ReVanced/revanced-discussions#171] [Question]
It would pretty sweet to be able to use an equalizer on YouTube (for listening to songs that aren't on Spotify for example).

**Comment by MulverineX** [comment]
You mean YouTube Music, right?

**Reply by guilhermequinta** [reply]
No, I'm talking about regular Youtube. Is there even any particular reason that regular YouTube doesn't support this? Seems like a no brainer to me... I listen to a lot of old vinyl rips and having to change to YouTube Music every time i want to listen with EQ is kinda of an unnecessary hassle in my opinion.
